### PR TITLE
On [Github] issues badge, add quotes around multi-word labels

### PR DESCRIPTION
**Pull request by platan**
Now "good first issue" badges in README works properly.

### CODE REVIEW

**Comment by paulmelnikow** [comment]
Thanks. I like the look of the label with quotes!

I mentioned in https://github.com/badges/shields/pull/1129/commits/d29bdea2bac0dc3aac7e7f77162169ad7709e83c that these badges were dependent on #1135 which has been merged but is waiting to be deployed. It's working correctly on the staging server:

https://shields-staging.herokuapp.com/github/issues/badges/shields/good%20first%20issue.svg

**Comment by platan** [comment]
Great to see them working. I've missed that change.

**Comment by paulmelnikow** [comment]
We should merge your issue -> pull request alt text change on L73.

Would you like to update the badge code to insert quotes into the badge title, when given a multi-word issue? In other words, to change **good first issue issues** to **“good first issue” issues**.

**Comment by tooomm** [comment]
>but is waiting to be deployed

Is there a new deploy scheduled? Or do you plan on merging specific PR's first?

File: service-tests/github.js [hunk 1]
```diff
@@ -84,10 +84,10 @@ t.create('GitHub open issues by label is > zero')
     value: Joi.string().regex(/^[1-9][0-9]*[kMGTPEZY]? open$/)
   }));
 
-t.create('GitHub open issues by label is > zero')
-  .get('/issues/Cockatrice/Cockatrice/Easy%20Change.json')
+t.create('GitHub open issues by multi-word label is > zero')
+  .get('/issues/Cockatrice/Cockatrice/App%20-%20Cockatrice.json')
   .expectJSONTypes(Joi.object().keys({
-    name: 'Easy Change issues',
```
Comment (platan) [inline service-tests/github.js]: Currently there are 283 open issues with "App - Cockatrice" label and 0 with "Easy Change" label.

**Comment by paulmelnikow** [comment]
> Is there a new deploy scheduled?

It's usually within 1–3 weeks. Thaddée is the only one with access, though we're aiming to add me sometime next month. You can monitor [the gh-pages branch](https://github.com/badges/shields/tree/gh-pages) which will update when the deploy happens.

> Or do you plan on merging specific PR's first?

Master is always ready to deploy, at least that's my aim.

**Comment by paulmelnikow** [comment]
Ran github tests locally with #1114. Two unrelated timeouts:

```
  1) Github Commits since by latest release
	[ GET http://localhost:1111/github/commits-since/microsoft/typescript/latest.json ]:
     Error: Timeout of 2000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.


  2) Github (pre-)Release
	[ GET http://localhost:1111/github/release/photonstorm/phaser/all.json ]:
     Error: Timeout of 2000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
```

**Comment by paulmelnikow** [comment]
Thanks! This is sweet! 🍰

**Comment by platan** [comment]
Thanks!

**Pull request merged**